eng/CodeGeneration.targets

```diff
@@ -18,6 +18,12 @@
     <_GenerateCode Condition="'$(_SupportsCodeGeneration)' == 'true' AND '$(AutoRestInput)' != ''">true</_GenerateCode>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TemporaryUsePreviousGeneratorVersion)' == 'true'">
```

Comment:
Nit: more grammatical to call it "TemporarilyUsePreviousGeneratorVersion", but...

Why not just let projects specify which version to use? At some point, breaking changes in the generator may be necessary, or we more some library to servicing-only. It would be safer to lock to a specific version, much like package managers allow locking to a fixed version (or short version range) for packages.

Reply:
>Why not just let projects specify which version to use? At some point, breaking changes in the generator may be necessary, or we more some library to servicing-only. It would be safer to lock to a specific version, much like package managers allow locking to a fixed version (or short version range) for packages.

Supporting multiple version of generator at the same time would be very costly and I want to avoid it as much as possible. This flag is temporary until we can get back to a unified version across all the projects.